packages/headless/src/modules/user-auth/controllers/profile.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, Get, UseGuards } from '@nestjs/common';
+import { ApiBearerAuth, ApiTags } from '@nestjs/swagger';
+import { UserId, UserInfo } from '../decorator/user.decorator';
+import { IJwtPayload } from '../interface/user.interface';
+import { JwtAuthGuard } from '../passport/jwt-auth.guard';
+import { UserProfileService } from '../services/user-profile.service';
+
+@ApiTags('User Profile API')
+@ApiBearerAuth()
+@UseGuards(JwtAuthGuard)
+@Controller('user-info')
+export class UserProfileController {
+  constructor(private userProfileService: UserProfileService) {}
+
+  @Get()
+  async getInfo(@UserId() userId: string) {
+    return await this.userProfileService.getProfile(userId);
+  }
+
+  @Get('user-by-id')
```

Comment:
Is this api path okay with the old project?

Reply:
resolved